lib/praxis/action_definition.rb

```diff
@@ -43,19 +46,39 @@ def initialize(name, resource_definition, **opts, &block)
         end
       end
 
-      resource_definition.action_defaults.each do |defaults|
-        self.instance_eval(&defaults)
+      version = resource_definition.version
+      version_prefix = resource_definition.version_prefix
+
+      api_info = ApiDefinition.instance.info(resource_definition.version)
+      
+      version_prefix = "#{api_info.base_path}#{version_prefix}"
+
+      prefix = Array(resource_definition.prefix)
```

Comment:
Except `version_prefix` isn't expected to be an array that any child resources could append to, it's passed to RoutingConfig as the `base`, which is the mandatory portion of the routing.